.goreleaser.yaml

```diff
@@ -65,107 +65,107 @@ dockers:
       - "--label=org.opencontainers.image.created={{ time \"2006-01-02T15:04:05Z07:00\" }}"
     image_templates:
-      - "ghcr.io/opentffoundation/opentf:{{ .Version }}-386"
+      - "ghcr.io/opentofu/opentofu:{{ .Version }}-386"
 
 docker_manifests:
-  - name_template: ghcr.io/opentffoundation/opentf:{{ .Version }}
+  - name_template: ghcr.io/opentofu/opentofu:{{ .Version }}
     image_templates:
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-amd64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-386
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-amd64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-386
 
-  - name_template: ghcr.io/opentffoundation/opentf:{{ .Major }}.{{ .Minor }}
+  - name_template: ghcr.io/opentofu/opentofu:{{ .Major }}.{{ .Minor }}
     image_templates:
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-amd64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-386
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-amd64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-386
 
-  - name_template: ghcr.io/opentffoundation/opentf:{{ .Major }}
+  - name_template: ghcr.io/opentofu/opentofu:{{ .Major }}
     image_templates:
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-amd64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-386
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-amd64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-386
 
-  - name_template: ghcr.io/opentffoundation/opentf:latest
+  - name_template: ghcr.io/opentofu/opentofu:latest
     image_templates:
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-amd64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm64
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-arm
-      - ghcr.io/opentffoundation/opentf:{{ .Version }}-386
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-amd64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm64
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-arm
+      - ghcr.io/opentofu/opentofu:{{ .Version }}-386
 
 nfpms:
   - file_name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}'
-    maintainer: 'https://github.com/orgs/opentffoundation/teams/opentf-core-team'
-    homepage: https://opentf.org
+    maintainer: 'https://github.com/orgs/opentofu/teams/opentf-core-team'
```

Comment:
I think you should ask an admin to change the team's name and add it here already

Reply:
@yaronya currently this waits, let's push it forward and update it later

Reply:
Can you create an issue for it please, and assign it to you? 🙏

Reply:
#527 + #528